agent/docker-compose.yml

```diff
@@ -27,7 +27,7 @@ services:
       - citamon-agent-net
 
   citamon_agent_rabbitmq_exporter:
-    image: kbudde/rabbitmq-exporter:1.0.0-RC
+    image: citamon/agent-rabbitmq-exporter:1.0.0-RC
```

Comment:
we need to get repo "citamon/agent-rabbitmq-exporter" source opened, put it into a sub dir